src/main/java/com/airdnb/clone/domain/stay/entity/AvailableAmenity.java

```diff
@@ -34,7 +34,7 @@ public class AvailableAmenity {
     @JoinColumn(name = "STAY_ID", foreignKey = @ForeignKey(name = "FK_STAY_AVAILABLE_ID"))
     private Stay stay;
 
-    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.�ALL)
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```

제가 삽질한 부분이죠.. 이렇게 되야합니다..